src/screens/Profile/components/Header/Header.js

```diff
@@ -34,56 +35,58 @@ export default class Header extends Component {
 
   render() {
     return (
-      <ResponsiveContext.Consumer>
-        {size => (
-          <Box
-            responsive
-            direction={getDirection(size)}
-            align="center"
-            justify="start"
-            pad={{ horizontal: "medium", vertical: "small" }}
-            gap="medium"
-          >
-            <Box responsive>
-              <PageTitle level={1} size="large" weight="bold">
-                Github Reporter
-              </PageTitle>
-            </Box>
+      <Slide left>
```

Comment:
I think that changing to `top` is better.

```suggestion
      <Slide top>
```